wandb/sdk/artifacts/storage_handlers/azure_handler.py

```diff
@@ -151,6 +148,13 @@ def store_path(
                         ),
                     )
                 )
+
+        if len(entries) > max_objects:
+            raise ValueError(
+                f"Exceeded {max_objects} objects tracked, pass max_objects to "
+                f"add_reference"
+            )
+
```

Comment:
we shouldn't ever hit this given the check above, all other handlers also have checks even though they explicitly limit the number of objects they get, so I kept this in to match that pattern

Reply:
just removed this anyway, we should never run into this